app/src/main/java/ru/zidkov/secondtask/MainViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package ru.zidkov.secondtask
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class MainViewModel : ViewModel() {
+    private val stateLiveMutable: MutableLiveData<MainState> = MutableLiveData<MainState>()
+    val stateLive: LiveData<MainState> = stateLiveMutable
```

Comment:
Чтобы лишнюю память не занимать
```suggestion
    val stateLive: LiveData<MainState> get() = stateLiveMutable
```